src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -154,18 +167,17 @@ In this section we show that smooth functions valued in a vector space `M` over
 field `𝕜` inherit a vector space structure.
 -/
 
-instance smooth_map_has_scalar
+instance has_scalar
   {V : Type*} [normed_group V] [normed_space 𝕜 V] :
   has_scalar 𝕜 C^∞⟮I, N; 𝓘(𝕜, V), V⟯ :=
 ⟨λ r f, ⟨r • f, smooth_const.smul f.smooth⟩⟩
 
 @[simp]
-lemma smooth_map.coe_smul
-  {V : Type*} [normed_group V] [normed_space 𝕜 V] (r : 𝕜) (f : C^∞⟮I, N; 𝓘(𝕜, V), V⟯) :
+lemma scoe_mul {V : Type*} [normed_group V] [normed_space 𝕜 V]
```

Comment:
This one should be `coe_smul`, not `scoe_mul`.

Reply:
```suggestion
lemma coe_smul {V : Type*} [normed_group V] [normed_space 𝕜 V]
```